internal/cmd/run/run.go

```diff
@@ -122,6 +123,7 @@ func Command() *cobra.Command {
 	sc.String("tunnel-owner", "sauce::tunnel::owner", "", "Sets the sauce-connect tunnel owner to be used for the run.")
 	sc.String("runner-version", "runnerVersion", "", "Overrides the automatically determined runner version.")
 	sc.String("sauceignore", "sauce::sauceignore", ".sauceignore", "Specifies the path to the .sauceignore file.")
+	sc.String("root-dir", "rootDir", ".", "Specifies the test directory")
```

Comment:
Just a tad bit more clarifying:
```suggestion
	sc.String("root-dir", "rootDir", ".", "Specifies the project directory. Not applicable to mobile frameworks.")
```